jax/experimental/jax2tf/jax_export.py

```diff
@@ -759,13 +786,17 @@ def check_sharding(op: ir.Operation, loc: ir.Location):
 
   def check_op(op: ir.Operation):
     op_name = op.operation.name
+
+    if not allowed_operation_names_re.match(op_name):
```

Comment:
@GleasonK Is there a better way to get the dialect name for the op than parsing its name?